### DISCUSSION TITLE
QSOs for my /P operations not uploading to eQSL

### DISCUSSION THREAD

**Discussion by M5JFS** [magicbug/Cloudlog#1002] [General]
QSOs made on my /P station profiles aren't uploading to eQSL for some reason.  Some did in the past (last year) but not now. 

I have set my eQSL QTH Nickname in my various /P station profiles and the QSOs appear in the list in cloudLog when I select eQSL QSO Upload.  The upload confirmation page in CloudLog shows no comment in the Result column against those QSOs after the upload has been performed.   

There are also a handful of old QSOs under my main station profile as well that never seem to get uploaded either for some odd reason.

Any thoughts on why this might be please?

**Comment by magicbug** [comment]
Odd, not sure the reason, will need to look at the code, guessing they just arent uploaded and sit there.

Shant mention the QTH Nicknames sounds like you have checked that and guessing they are all under the same eQSL account.

**Reply by M5JFS** [reply]
yes, I have 4 /P profiles all with the same eQSL nickname which matches my M5JFS/P profile nickname in eQSL itself.  I have a few on my main station profile which also don't seem to ne uploading.  I might take a look at the tables and see if there's anything peculiar about the data on those records compared to others that upload ok

**Reply by magicbug** [reply]
Is your eQSL login the M5JFS/P profile?

**Reply by M5JFS** [reply]
![image](https://user-images.githubusercontent.com/73586719/115998997-df72cf00-a5e1-11eb-904f-0c41a1fdff97.png)
![image](https://user-images.githubusercontent.com/73586719/115999017-faddda00-a5e1-11eb-8bd0-b50b2c593ab7.png)
![image](https://user-images.githubusercontent.com/73586719/115998958-b9e5c580-a5e1-11eb-82cf-93a8a045961d.png)

**Reply by M5JFS** [reply]
I have changed the eQSL userid field in my CloudLog M5JFS account to be m5jfs/p and it has now uploaded the QSOs for my portable ops, but it won't download matching eQSLs - giving the message "No such Username/Password found This could mean the wrong callsign or the wrong password, or the user does not exist."

**Reply by M0TZO** [reply]
From your images, the Nicknames you need to add to each Profile are "_Home QTH_" and "_Portable IO90_" not M5JSF or M5JFS/P.
